src/helpers/getMovieUtils.ts

```diff
@@ -17,31 +17,52 @@ const getMoviesByName = async (name: string): Promise<MovieData> => {
  * This function is ran for specific <ShowCard /> data with a movieID.
  * @returns {Promise<MovieDetailsData>} | Specific data for a movie that is not originally supplied by getMoviesByName.
  */
-const getMovieDetails = async (id: number): Promise<MovieDetailsData> => {
+const getMovieDetails = async (id: number): Promise<ShowData> => {
     const response = await fetch(
         `https://api.themoviedb.org/3/movie/${id}?api_key=${
             import.meta.env.VITE_MOVIEDB_KEY
         }&append_to_response=images,release_dates`
     );
-    return response.json() as Promise<MovieDetailsData>;
+    const data = (await response.json()) as MovieDetailsData;
+    const returnRating = (arr: MovieDetailsData) => {
+        for (let i = 0; i < arr.release_dates.results.length; i++) {
+            if (arr.release_dates.results[i].iso_3166_1 === 'US') {
+                return arr.release_dates.results[i].release_dates[0].certification;
+            }
+        }
+        return 'No rating available';
+    };
+    const obj: ShowData = {
+        id: data.id,
+        poster_path: data.poster_path,
+        title: data.original_title,
+        release_date: data.release_date,
+        age_rating: returnRating(data),
+        runtime: data.runtime,
+        vote_average: data.vote_average,
+        vote_count: data.vote_count,
+        overview: data.overview,
+    };
+    console.log(JSON.stringify(obj));
```

Comment:
```suggestion

```

Reply:
we could also outright return the object here rather than creating a var, like the TV file.

Reply:
Yeah I needed to console log the var to copy its content so It could be placed in the new tests json file. I forgot about it lol. Ill fix it

Reply:
haha I figured, all good! We could configure es-lint to warn about logs, just to remind us cause it is so easy to forget.